recipes/marisa-trie/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "marisa-trie" %}
```

Comment:
It seems there is a C++ library with the same name. Should we prefix this with `python-`?